packages/eslint-config-terra/UpgradeGuide.md

```diff
@@ -0,0 +1,20 @@
+# Eslint Config Terra Upgrade Guide
+
+## Changes from version 3 to version 4
+
+### Project Name Updated with @cerner Scope
+
+The project will need to be updated from eslint-config-terra to @cerner/eslint-config-terra in the devDependency list in package.json. The eslint config then becomes:
+
+```diff
+{
+  "eslintConfig": {
+-    "extends": "terra"
++    "extends": "@cerner/terra"
+  },
+}
+```
```

Comment:
```suggestion
```

### Upgrade Node

This version of eslint-config-terra drops support for Node 8.x.x. Projects will need to be upgraded to at least Node 10.
```